File: class11/cpp/AP1111-CalculoConsumoCombustivel.cpp
#include <iostream>
#include <iomanip>
int main() {
    double km, litros;
    std::cout << "Distância percorrida (km): "; std::cin >> km;
    std::cout << "Combustível gasto (L): "; std::cin >> litros;
    std::cout << "Consumo médio: " << std::fixed << std::setprecision(1) << km/litros << " km/L" << std::endl;
    return 0;
}

File: class11/java/AP1111-CalculoConsumoCombustivel.java
import java.util.Scanner;
public class CalculoConsumoCombustivel {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Distância percorrida (km): ");
        double km = sc.nextDouble();
        System.out.print("Combustível gasto (L): ");
        double litros = sc.nextDouble();
        System.out.printf("Consumo médio: %.1f km/L%n", km/litros);
        sc.close();
    }
}